Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM ubuntu:16.04
+MAINTAINER Devin Ekins <devops@keen.io>
+
+# Setup the dependencies
+RUN apt-get update -y
+RUN apt-get install -y apt-utils
+RUN apt-get install -y libtool make automake
+
+# Copy local repo into the container
+ADD . /twemproxy
+WORKDIR /twemproxy
+
+# Install Twemproxy
+RUN autoreconf -fvi && \
+    ./configure --enable-debug=full && \
+    make
+
+# Expose Twemproxy Ports
+EXPOSE 22122 22222
+
+# Start Twemproxy
+# Use default configuration file location/name.
+# Use mbufsize from ansible commandline variable.
+ENTRYPOINT src/nutcracker
```

Comment:
Nit: we usually put these in the exec form format (which is preferred) like so: `ENTRYPOINT ["src/nutcracker"]`